securedrop/journalist_templates/admin_add_user.html

```diff
@@ -6,17 +6,39 @@
 
 <form method="post">
   <input name="csrf_token" type="hidden" value="{{ csrf_token() }}">
-  <input name="password" type="hidden" value="{{ password }}">
-  <p><input name="username" type="text" placeholder="{{ gettext('Username') }}"></p>
+  <p>
+    {{ form.password(value=password, id=False) }}
+    <br>
+    {% for error in form.password.errors %}
+      <span class="form-validation-error">{{ error }}</span>
+    {% endfor %}
+  </p>
+
+  <p>
+    {{ form.username(placeholder=gettext('Username')) }}
```

Comment:
**id=False** ? I don't see why it would hurt to have an id but ... since the password above has **id=False** this would be consistent don't you think ?

Reply:
yep, I generally think it's good to have `id` defined for functional testing, I just set the password `id` to false above in the one field as by default WTForms would use `id="password"` which would create a duplicate `id` (`id="password"` is used elsewhere in the file).